### ISSUE TITLE
better documentation, or reference, regarding GAEN

### ISSUE THREAD

**Issue by marado** [DP-3T/dp3t-sdk-backend#197]
The README states that "It is recommended to adopt the new Exposure Notification format, since support on clients is much better", however nothing is said about how is it "much better".

**Comment by ineiti** [comment]
If I understand correctly, this has to do with the following:
- `DP3T` BLE format is a 'generic' format that every app can create / use, so:
  - it is much more engergy-hungry, as the app needs to run to support it
  - more difficult to standardize, as different apps might implement it differently
- `GAEN` is based on the `DP3T` protocol, but standardized by google and apple:
  - it can work independently of the app and uses much less energy
  - it gives an additional layer of security as it doesn't allow the app to log all BLE messages
  - one negative point is that it only has a pre-defined way of calculating the exposure-risk

**Comment by marado** [comment]
> it gives an additional layer of security as it doesn't allow the app to log all BLE messages

This, in my opinion, is quite debatable. An app with a Free Software implementation gives us that additional layer of security: we can audit the app, and make sure it is not logging what is should not. The use of `GAEN`, however, cannot give us any such guarantee: on the contrary, not only it is developed and updated in a closed fashion, but its closed source nature makes it inauditable.

**Comment by ineiti** [comment]
Yes, this is correct and a shame. Unfortunately I think this is currently the best we can have.

I was referring to the fact that the apps don't have an easy way to get to a full list of `Rolling Proximity Identifiers` that the phone received. Which makes it a little bit harder to abuse the exchanged BLE tokens for tracing other people.

I did a small test with an app that shows all BLE tokens, and I couldn't detect any RPIs that the app sees on my android phone. So either I looked wrongly, or the GAEN filters the BLE messages and only lets non-RPIs through... To be confirmed...
